desktop/components/main_layout/footer/view.coffee

```diff
@@ -28,7 +27,7 @@ module.exports = class FooterView extends Backbone.View
 
   show: ->
     @$el.show()
-    reInitRVARail(@$recentlyViewedArtworks) if shouldShowRVARail()
+    reInitRVARail(@$recentlyViewedArtworks) if shouldShowRVARail() && @$recentlyViewedArtworks.length > 0
```

Comment:
I noticed this crash b/c it's initialized as part of the main layout asset package. But in the case of the blank main layout template, this element isn't here. /shrug